inference-engine/src/cldnn_engine/cldnn_program.cpp

```diff
@@ -177,9 +178,15 @@ std::shared_ptr<cldnn::program> Program::BuildProgram(const std::vector<std::sha
                                                       bool createTopologyOnly) {
     OV_ITT_SCOPED_TASK(itt::domains::CLDNNPlugin, "Program::BuildProgram");
     cldnn::build_options options;
+    auto debug_config = cldnn::debug_configuration::get_instance();
     if (!m_config.graph_dumps_dir.empty()) {
         options.set_option(cldnn::build_option::graph_dumps_dir(m_config.graph_dumps_dir));
     }
+
+    GPU_DEBUG_IF(debug_config != nullptr && !debug_config->dump_graphs.empty()) {
```

Comment:
It looks like we don't need `debug_config != nullptr` check under `GPU_DEBUG_IF`

Reply:
Thanks for the comment, I fixed it.